Find a functionally equivalent code:android.graphics.Matrix.preConcat:COMMENT
Method Modifier: public      
Comment:/**
 * Preconcats the matrix with the specified matrix.
 * M' = M * other
 */

@@ -1,3 +1,4 @@
 {
-    return native_preConcat(native_instance, other.native_instance);
+    native_preConcat(native_instance, other.native_instance);
+    return true;
 }

